api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
-<h3>Medlemsinformation</h3>
-<p>Namn: <i>{{member.firstname}} {{member.lastname}}</i><br/>
-    Email: <i>{{member.email}}</i><br/>
-    Medlemsnummer: <i>{{member.member_number}}</i><br/>
-</p>
-
-<p>
-    Om föreningen behöver nå dig så använder vi den här emailen. Det är därför viktigt att du tar ansvar för att den här e-mailadressen fungerar, eller anmäler ny email. Medlemsnummret används för vår interna organisation och du kommer att behöva det för att anmäla dig till våra kurser.
-</p>
+<meta charset="UTF-8">
+<style>
+    .text-box {
+        border: 1px solid #000;
+        padding: 5px;
+    }
+  </style>
+
+{% block language clarification %}
+<div class="text-box">
+    <h3 > About language</h3>
+    <p>The primary language on Stockholm Makerspace is Swedish, but you are very welcome to use English as well. Newsletters and important announcements as well as our courses are as a rule available in English. If you need help to translate something from Swedish to English, don’t hesitate to ask other members or post about it on Slack.</p>
+</div>
+<hr>
+{% enbblock %}
 
-<h3>TVÅ TYPER AV MEDLEMSKAP</h3>
-<p>
-    <b>Grundmedlem</b><br/>
-    Som medlem i föreningen så är du välkommen på kurser och workhops som vi arrangerar, samt årsmötet. Vill du utföra självständigt arbete i lokalen så ska du skaffa makerspace access.
-</p>
+{% block swedish %}
+<h2>SVENSKA</h2>
+<p>Hejsan! Välkommen till Stockholm Makerspace! &#128515;<br> Vi hoppas du uppskattade introkursen och är redo att påbörja din kreativa resa med oss.</p>
 
-<p>
-    <b>Makerspace Access</b><br/>
-    För medlemmar som har Makerspace Access så är lokalen öppen dygnet runt, året runt, men för att komma in första gången så måste du besöka en medlemsintroduktion. Medlemsintroduktioner äger rum ungefär varannan vecka, och datum hittar du i vårt <a href="https://makerspace.se/kalendarium">kalendarium</a>. För få access måste du skriva under ett avtal där du ålägger dig att följa våra <a href="https://wiki.makerspace.se/Lokalregler">lokalregler</a>, samt visa ID.
-</p>
+<p>Som ny medlem har du förmodligen en del frågor. I detta mail kommer du få en sammanfattning av vad du fick höra på introkursen och länkar till några onlineresurser som hjälper dig att sätta igång.</p>
 
-<h3>Jag vill gå kurs! När är det kurs?</h3>
-<p>All verksamhet bedrivs av volontärer, som du och jag. Vi hjälps åt att lära varandra maskinerna så att alla kan utföra självständigt arbete. Man delar vad man kan, och ber om hjälp om man kör fast. Några saker måste man gå kurs för att kunna använda, till exempel: svetsrummet, laserskäraren, brodyrmaskinen och resin-printern. Kurser annonseras i <a href="https://www.facebook.com/groups/makerspace.se">Facebookgruppen</a> och på <a href="https://wiki.makerspace.se/Slack">Slack</a> och läggs in i kalendern. Det finns ingen kö, först till kvarn gäller.
+<h3>Dina medlemsuppgifter:</h3>
+<p>Namn: {{member.firstname}} {{member.lastname}}<br/>
+    Epost: {{member.email}}<br/>
```

Comment:
Epost är implicit eftersom det är dit mailet kom. Let's not make the email longer than we need to.

Reply:
I understand your thinking.
We took this section from the previous version of the email and, I believe, the purpose of this section is to let know the new members that the email address is important information that we have and that they can change it by emailing info@makerspace.se.
In any case, this version of the email is significantly shorter than the one before.  That being said, I don´t have strong feelings about this point 😊. I can take it out if you still feel is unnecessary.

BTW, thank you so much for taking the time to look at this. Really appreciate it. 👏